### ISSUE TITLE
Two mAp value tables are being printed in the terminal while training the algorithm.

### ISSUE THREAD

**Issue by vijay2249** [mikwieczorek/centroids-reid#30]
While looking into the code, one of the arguments that are used is SOLVER.EVAL_PERIOD
Why and how is this useful in printing the output?

While training the algorithm over a dataset, we are getting two outputs that displays two different mAp values for each time
![test-results](https://user-images.githubusercontent.com/65058877/203803996-ef4bfb6d-4a48-45fc-9d10-f81dbb0a2421.jpeg)

**Comment by LucasMaris** [comment]
I'm also confused about this. This also seems to be the case for the tfevent files saved in the logs, as they include duplicate entries for mAP & Top-K values after 119 training epochs, the first set of values being much lower than the latter. My guess is that the first set are testing results and the second set training results? And the opposite on your screenshot? It would be nice to get more info on this.

**Comment by vijay2249** [comment]
@LucasMaris 
Actually we figured this issue and what the values are.

The first one which is shown in the above figure are the ones with accuracy prediction when the algorithm is trained with centroid loss which is mentioned the readme.
The second one in the figure which has less mAP value than the first one is the prediction accuracy results while training algorithm without centroid-loss function which results less accuracy.
Hence the need to use centroid-loss to improve the accuracy..
